bindings/python/test/curve_fitting/interpolator/test_cubic_spline.py

```
@@ -43,3 +43,17 @@ def test_evaluate(self):
             assert pytest.approx(interpolator.evaluate(i * 10.0)) == y[i]
 
         assert pytest.approx(interpolator.evaluate(np.linspace(0.0, 90.0, 10))) == y
+
+    def test_compute_derivative(self):
+        interpolator = CubicSpline(
+            x=[0.0, 1.0, 2.0, 3.0, 4.0, 5.0], y=[0.0, 3.0, 6.0, 9.0, 17.0, 5.0]
+        )
+
+        assert interpolator.compute_derivative(1.5) is not None
+
+    def test_compute_derivative(self):
```

Ruff (0.8.2)

Redefinition of unused `test_compute_derivative` from line 47

(F811)